www/app/App.js

```diff
@@ -36,22 +34,8 @@ export const App = ({ Component, pageProps }) => {
                 <meta name="msapplication-TileColor" content="#ffffff" />
                 <meta name="msapplication-config" content="/favicons/browserconfig.xml?v=M4KN2GElyG" />
                 <meta name="theme-color" content="#ffffff" />
-                {/* Facebook & search engines */}
-                <meta property="og:url" content={ SEO_DATA.url } />
-                <meta property="og:type" content="website" />
-                <meta property="og:title" content={ SEO_DATA.title } />
-                <meta property="og:description" content={ SEO_DATA.description } />
-                <meta property="og:site_name" content={ SEO_DATA.title } />
-                <meta property="og:image" content={ SEO_DATA.image.src } />
-                <meta property="og:image:width" content={ SEO_DATA.image.width } />
-                <meta property="og:image:height" content={ SEO_DATA.image.height } />
-                {/* Twitter */}
-                <meta property="twitter:card" content="summary_large_image" />
-                <meta property="twitter:url" content={ SEO_DATA.url } />
-                <meta property="twitter:title" content={ SEO_DATA.title } />
-                <meta property="twitter:description" content={ SEO_DATA.description } />
-                <meta property="twitter:image" content={ SEO_DATA.image.src } />
             </Head>
+            <Seo data={ seoData } />
```

Comment:
If I'm not mistaken the `Seo` component should still be placed inside `Head`. Personally, I'd prefer to have it right after the opening `Head` tag

Reply:
It renders the `Head` tag so it shouldn't be inside it: https://github.com/moxystudio/next-seo/blob/master/src/Seo.js#L35